charts/ark-alfresco/charts/ark-alfresco-activemq/Chart.lock

```diff
@@ -0,0 +1,6 @@
+dependencies:
```

Comment:
I see many other .lock files were deleted, and so far this is the only added .lock file, did you mean to add it?

Reply:
nvm I see the other ones now.

Reply:
They get added automatically by Helm. They're for dependency management so they're rendered artifacts. However, they do need to be part of the chart when packaged, so they have to remain as part of the source control package.